modules/currency.js

```diff
@@ -0,0 +1,206 @@
+
+import bidfactory from 'src/bidfactory';
+import { STATUS } from 'src/constants';
+import { ajax } from 'src/ajax';
+import * as utils from 'src/utils';
+import bidmanager from 'src/bidmanager';
+
+$$PREBID_GLOBAL$$.currency = setConfig;
+
+const DEFAULT_CURRENCY_RATE_URL = "http://prebid.aws.rubiconproject.com/latest.json";
+const CURRENCY_RATE_PRECISION = 4;
+
+var bidResponseQueue = [];
+var conversionCache = {};
+var currencyRatesLoaded = false;
+var adServerCurrency = 'USD';
+
+var oldBidResponse;
```

Comment:
can you make this name more descriptive? I was confused about what it was until I saw line 51.